src/store/main.ts

```diff
@@ -12,6 +12,7 @@ import { getUserAvatarPlaceholder } from './lib';
 
 export const queryLimitTribes = 100;
 export const queryLimit = 10;
+export const orgQuerLimit = 9000000000;

```

Comment:
I'm not sure this is a reasonable amount haha maybe we can find a better way than just setting a large limit